google/cloud/bigtable/table.py

```diff
@@ -558,7 +566,7 @@ def read_row(self, row_key, filter_=None):
         """
         row_set = RowSet()
         row_set.add_row_key(row_key)
-        result_iter = iter(self.read_rows(filter_=filter_, row_set=row_set))
+        result_iter = iter(self.read_rows(filter_=filter_, row_set=row_set, retry=retry))
```

Comment:
Thanks for submitting this

I can't merge due to a lint error. i think it's because this line is too long. Maybe break it up:

```
        result_iter = iter(
            self.read_rows(filter_=filter_, row_set=row_set, retry=retry)
        )

```

Reply:
@daniel-sanche done https://github.com/googleapis/python-bigtable/commit/39e8f90b12ab14cd7981bf7f1f1d80caa9a1447d